### ISSUE TITLE
[bug] api/generate -  Gateway Timeout 504 

### ISSUE THREAD

**Issue by HBO30** [ddiu8081/chatgpt-demo#153]
### Describe the bug

as normal, I just send hi 

![image](https://user-images.githubusercontent.com/72744132/223291787-0dfee35d-50fb-407d-9932-9534d41c47c8.png)

---
then too much time pending 

![image](https://user-images.githubusercontent.com/72744132/223292099-a2a6da10-1aac-453f-9f3f-b7b1d01944b3.png)

---
then error 504 

![image](https://user-images.githubusercontent.com/72744132/223292344-f33df20f-0b84-418e-acb8-31b907f8bb97.png)
![image](https://user-images.githubusercontent.com/72744132/223292752-2613c1bf-ddfa-4246-8457-ff428435b97c.png)

with  message 
>An error occurred with your deployment
>EDGE_FUNCTION_INVOCATION_TIMEOUT

## Request 
```bash
curl 'https://chatgpt.ddiu.me/api/generate' \
  -H 'authority: chatgpt.ddiu.me' \
  -H 'accept: */*' \
  -H 'accept-language: en,en-US;q=0.9,ar;q=0.8' \
  -H 'content-type: text/plain;charset=UTF-8' \
  -H 'cookie: mp_10bab2a6cec3a2ef62083257f3f09083_mixpanel=%7B%22distinct_id%22%3A%20%22186b453fe1d317-0fdc99e75b6021-26031951-100200-186b453fe1e29d%22%2C%22%24device_id%22%3A%20%22186b453fe1d317-0fdc99e75b6021-26031951-100200-186b453fe1e29d%22%2C%22%24initial_referrer%22%3A%20%22%24direct%22%2C%22%24initial_referring_domain%22%3A%20%22%24direct%22%7D' \
  -H 'origin: https://chatgpt.ddiu.me' \
  -H 'referer: https://chatgpt.ddiu.me/' \
  -H 'sec-ch-ua: "Chromium";v="110", "Not A(Brand";v="24", "Google Chrome";v="110"' \
  -H 'sec-ch-ua-mobile: ?0' \
  -H 'sec-ch-ua-platform: "Windows"' \
  -H 'sec-fetch-dest: empty' \
  -H 'sec-fetch-mode: cors' \
  -H 'sec-fetch-site: same-origin' \
  -H 'user-agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/110.0.0.0 Safari/537.36' \
  --data-raw '{"messages":[{"role":"user","content":"hi"}],"time":1678154495751,"sign":"63de47363334984585a7f91cf64a4fa71d3d3463f3cd29e6741f659c15d3aa3c"}' \
  --compressed
```

## Response 
```
An error occurred with your deployment

EDGE_FUNCTION_INVOCATION_TIMEOUT

```

### Reproduction

https://chatgpt.ddiu.me/

### System Info

```Shell
os : Windows 10
brwoser : Chrome
```

### Used Package Manager

npm

### Validations

- [X] Follow our [Code of Conduct](https://github.com/ddiu8081/.github/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://github.com/ddiu8081/.github/blob/main/CONTRIBUTING.md).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A, please open a GitHub Discussion instead.
- [X] The provided reproduction is a [minimal reproducible](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by h7ml** [comment]
The possible reason for this could be that the OpenAI quota has been exhausted due to excessive usage by too many people.

**Comment by SakigamiYang** [comment]
这是什么bug啊，不知道接口域名喜提GFW名单了吗

**Comment by Root-FTW** [comment]
> The possible reason for this could be that the OpenAI quota has been exhausted due to excessive usage by too many people.



> The possible reason for this could be that the OpenAI quota has been exhausted due to excessive usage by too many people.

Does it affect everyone who has chatgpt APi?

I put the mount ne vercel I put the my api and it worked only a couple of hours after that, then I got the error.

I deleted my API and created a new one and still the same error, I changed my subdomain in vercel, changed my IP and still the error, I tried on other devices and the error is the same.

Does anyone have any solution? :(
